src/atoms/Link/Link.web.js

```diff
@@ -43,27 +50,12 @@ Link.propTypes = {
   href: PropTypes.string,
   target: PropTypes.oneOf(['_blank', '_self', '_parent', '_top']),
   testID: PropTypes.string,
-  rel: PropTypes.oneOf([
-    'alternate',
-    'author',
-    'bookmark',
-    'external',
-    'help',
-    'license',
-    'next',
-    'nofollow',
-    'noreferrer',
-    'noopener',
-    'prev',
-    'search',
-    'tag',
-  ]),
+  rel: PropTypes.string,
 };
 
 Link.defaultProps = {
   size: 'medium',
   href: '#',
-  target: '_self',
```

Comment:
Why removing this?

Reply:
removed this, otherwise, target will always be defined.